### PR TITLE
landing page copy update

### PR DESCRIPTION
**Pull request by elena-vi**
Updated copy on landing page to reflect new designs

screenshot-localhost_3000-2021 10 11-18_16_15

### CODE REVIEW

**Comment by elena-vi** [comment]
[Landing Page (MVP)](https://trello.com/c/TTnaXdE8/163-landing-page-mvp)

File: src/pages/landingPage.js [hunk 1]
```diff
@@ -19,37 +16,75 @@ import { Fragment } from 'react';
...
+    <Link href="https://www.lincoln.gov.uk/contact-1/get-touch">
+      contact details page
+    </Link>
+  )
+
   return (
     <Fragment>
       <GridRow>
         <GridCol setWidth="two-thirds">
           <H1>Report a repair</H1>
```
Comment (Bhavesh-Patel) [inline src/pages/landingPage.js:29]: Title from [prototype](https://southwark-repairs-alpha-master.herokuapp.com/beta/v2/) is "Request a repair"
```suggestion
          <H1>Request a repair</H1>
```

File: src/pages/landingPage.js [hunk 2]
```diff
@@ -19,37 +16,75 @@ import { Fragment } from 'react';
 
 
 function LandingPage() {
-  const start=()=>{
-    window.location.hash='/report-repairs'
-  }
+  const contactDetailsPageLink = (
+    <Link href="https://www.lincoln.gov.uk/contact-1/get-touch">
```
Comment (Bhavesh-Patel) [inline src/pages/landingPage.js:20]: In the [prototype](https://southwark-repairs-alpha-master.herokuapp.com/beta/v2/) clicking the "contact details page" link opens the url in a new page.
```suggestion
    <Link href="https://www.lincoln.gov.uk/contact-1/get-touch" target="_blank">
```

**Review by Bhavesh-Patel — CHANGES_REQUESTED** [review]
Couple of small differences between the prototype and production.

**Review by Bhavesh-Patel — APPROVED** [review]

**Pull request merged**